packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -1214,6 +1226,24 @@ class FlutterPlugin implements Plugin<Project> {
     }
 }
 
+class AppLinkSettings {
```

Comment:
Can you extract these files like you did for [android_app_link_settings.dart](https://github.com/flutter/flutter/pull/131805/files#diff-fb73c485928218a3207e77e0384069a8bce9714333a0fecaba7f1f43d8ecde58)